integrationtest/initial_release/main.d

```diff
@@ -20,7 +20,7 @@ class InitialRelease : TestCase
 {
     this ( )
     {
-        super();
+        super(8001);
```

Comment:
Mmm, this is a bit fragile. At some point it might be good to find a way to automatically assign ports to tests (maybe some *mixin* magic?), or just serialize tests by making them interdependent.